content/webapp/utils/iiif/v3/index.ts

```diff
@@ -173,8 +173,12 @@ function getCanvasTextServiceId(canvas: Canvas): string | undefined {
   return textAnnotation?.id;
 }
 
-export const isChoiceBody = (item: IIIFItemProps): item is ChoiceBody => {
-  return typeof item !== 'string' && 'type' in item && item.type === 'Choice';
+export const isChoiceBody = (
+  item: IIIFItemProps | undefined
```

Comment:
Is there a wider issue with our types where this shouldn't have to potentially be `undefined`? (as in, if we use `isChoiceBody` it's because there's an item to pass in?)

Reply:
That makes sense, but definitely a bigger piece of work